CMakeLists.txt

```diff
@@ -44,12 +44,12 @@ add_library(effil SHARED ${SOURCES})
 target_link_libraries(effil ${LUA_LIBRARY})
 if (WIN32)
     set_target_properties(effil PROPERTIES
-        PREFIX lib
+        PREFIX ""
```

Comment:
In this case you can delete `PREFIX` from all CMakeList.txt

Reply:
Hmm, that sounds much better